ci-operator/config/openshift/os/openshift-os-master.yaml

```diff
@@ -1,5 +1,5 @@
 base_images:
-  coreos_coreos-assembler_48:
+  coreos_coreos-assembler_49:
```

Comment:
Seems like this should just be called `coreos_coreos-assembler_latest` instead, no? There is no 4.9 cosa yet so this is misleading.

Or actually, since we have to keep using the 4.8 image for now until we fix https://github.com/openshift/os/issues/552, maybe better to just not touch this file at all. (But then eventually once we do fix that, it'd be `coreos_coreos-assembler_latest` here and `tag: "latest"` below.)

Reply:
@jlebon meh, it got merged already. Let me know what do you think, if should I change it following the comments above.

Reply:
That's OK, I'm sure we'll keep tweaking these settings in the future as we sort everything out.

Thanks a lot for looking at this!